tests/base/src/test/java/org/keycloak/tests/admin/authz/fgap/AbstractPermissionTest.java

```diff
@@ -36,10 +36,12 @@
 
 public abstract class AbstractPermissionTest {
 
-    @InjectRealm(config = RealmAdminPermissionsConfig.class)
+    public final String REALM_REF = "AbstractPermissionTestRealm";
+
+    @InjectRealm(config = RealmAdminPermissionsConfig.class, ref = REALM_REF)
```

Comment:
Why are you changing this? These tests have a single realm, so shouldn't need to specify a realm-ref